diff -Naur mpqc-2.3.1.orig/bin/sc-config.in mpqc-2.3.1/bin/sc-config.in
--- mpqc-2.3.1.orig/bin/sc-config.in	2014-10-16 23:58:19.000000000 +0400
+++ mpqc-2.3.1/bin/sc-config.in	2014-10-16 23:58:46.000000000 +0400
@@ -45,7 +45,7 @@
 
 sclibs=""
 for i in $all_sclibs; do
-  if [ -f $libdir/$i ]; then
+  if [ -f $libdir/${i/la/so} ]; then
     li=`echo $i | sed "s/^lib\([a-zA-Z0-9]*\).$LIBSUF/-l\1/g"`
     sclibs="$sclibs $li"
   fi
